### DISCUSSION TITLE
Support for other G-suite apps

### DISCUSSION THREAD

**Discussion by bluescreen10** [zoidsh/meru#295] [Feature Requests] [answered]
Hi,

I wanted to discuss the possibility of adding support for other gsuite apps (calendar, drive, docs, etc.) within the same app. Today when you click a link for any other app it opens it up in the browser, but I think it'd be more useful to stay within the same window (see screenshots below).

<img width="1274" alt="Screen Shot 2021-10-06 at 1 59 28 PM" src="https://user-images.githubusercontent.com/86094/136267412-cca4e5bf-e803-4f90-94d3-ca40c0a6c082.png">
<img width="1274" alt="Screen Shot 2021-10-06 at 1 58 56 PM" src="https://user-images.githubusercontent.com/86094/136267333-04b852f8-4d25-4d69-9ccb-cb57f2b49724.png">

I've hacked this by using electron's BrowserView but before I polish the code and create a pull-request wanted to gather the interest of supporting this.

**Comment by timche** [comment, marked as answer]
Unfortunately Gmail Desktop is only focusing on Gmail and will stay that way, sorry.

**Reply by robmonte** [reply]
Just thought I'd show a little support for the idea here and mention that, from what I've seen, there's at least a sizeable desire for Calendar. I believe Gmail and Google Calendar are very tightly integrated and there's no well-made GCal desktop application that I've found. 

Rather than a big project integrating all of Gsuite into a single application like the original post outlined, I think even just a second standalone application for Google Calendar with some features that you put in gmail-desktop, like dark reader, would be amazing.

**Comment by patrickyan** [comment]
@bluescreen10 is this a functional app or a mockup?
